libs/react/src/lib/datepicker/datepicker.tsx

```diff
@@ -100,6 +121,8 @@ export const Datepicker = ({
     }
   }
 
+  console.log(props)
```

Comment:
Remove `console.log`?

Reply:
@vsjolander There were two different logs, one was not removed https://github.com/seb-oss/green/blob/main/libs/react/src/lib/datepicker/datepicker.tsx#L124